documentation/tests/test_documentation_admin.py

```diff
@@ -0,0 +1,39 @@
+from django.test import TestCase
+from django.contrib.auth.models import Permission
+from django.contrib.sites.models import Site
+
+from mezzanine.core.models import SitePermission
+
+from curricula.factories import UserFactory
+
+class DocumentationAdminTestCase(TestCase):
+    def setUp(self):
+        user = UserFactory()
+        user.is_staff = True
+        user.save()
+        self.user = user
+        self.assertTrue(self.client.login(username=user.username, password='password'))
+
+        site = Site.objects.first()
+        siteperms = SitePermission.objects.create(user=user)
+        siteperms.sites.add(site)
+
+    def render_add(self, type):
+        response = self.client.get('/admin/documentation/'+type+'/add/')
```

Comment:
it would be a bit more pythonic to construct these using string interpolation, as shown here: https://github.com/code-dot-org/curriculumbuilder/blob/e2a7c0ef6389791bd0558993652f4109c0bbd180/lessons/admin.py#L217-L217